src/clj/game/cards/operations.clj

```diff
@@ -1356,6 +1356,18 @@
                  (shuffle! state side :deck)
                  (draw state side eid (count targets) nil))}
 
+   "Standard Procedure"
+   {:req (req (last-turn? state :runner :successful-run))
+    :delayed-completion true
```

Comment:
This doesn't need delayed completion, because it does not trigger any awaitable actions. Did you try it with AccDiag and see an error?